apps/www/lib/translations.json

```diff
@@ -5412,6 +5488,10 @@
       "key": "Onboarding/Page/button",
       "value": "Zum Magazin"
     },
+    {
+      "key": "Onboarding/Page/climate/button",
+      "value": "Weiter ins Klimalabor"
+    },
     {
```

Comment:
In all other t9n strings we used "Zum Klimalabor". I am horrible but grammar, so this might be perfectly fine, but it seems a bit inconsistent to me.